src/tilemaker.cpp

```diff
@@ -193,6 +215,13 @@ int main(int argc, char* argv[]) {
 	Box clippingBox;
 	MBTiles mapsplitFile;
 	double minLon=0.0, maxLon=0.0, minLat=0.0, maxLat=0.0;
```

Comment:
maybe change this to std::vector<double> ? or struct of type bbox: struct bbox { double minLon; double maxLon; ... }. 
such that parsebbox can return this.

Reply:
The changes following your suggested change extend the scope of this pull request quite a lot. That's why I decided to skip this for the time being.